File: Serene/Serene.Script/Imports/FormContexts/Northwind.EmployeeTerritoryForm.cs

namespace Serene.Northwind
{
    using Serenity;
    using Serenity.ComponentModel;
    using System;
    using System.Runtime.CompilerServices;

    public partial class EmployeeTerritoryForm : PrefixedContext
    {
        [InlineConstant] public const string FormKey = "Northwind.EmployeeTerritory";
    
        public EmployeeTerritoryForm(string idPrefix) : base(idPrefix) {}
    
        public StringEditor TerritoryID { [InlineCode("{this}.w('TerritoryID', Serenity.StringEditor)")] get; private set; }
    }
}

File: Serene/Serene.Web/Modules/Common/Imports/ServerTypings/Northwind.EmployeeTerritoryForm.ts
namespace Serene.Northwind {
    export class EmployeeTerritoryForm extends Serenity.PrefixedContext {
        static formKey = 'Northwind.EmployeeTerritory';

    }

    export interface EmployeeTerritoryForm {
        TerritoryID: Serenity.StringEditor;
    }

    [['TerritoryID', Serenity.StringEditor]].forEach(x => Object.defineProperty(EmployeeTerritoryForm.prototype, <string>x[0], { get: function () { return this.w(x[0], x[1]); }, enumerable: true, configurable: true }));
}

